jda/src/main/java/co/aikar/commands/JDACommandPermissionResolver.java

```diff
@@ -24,6 +24,10 @@ public boolean hasPermission(JDACommandEvent event, String permission) {
             return false;
         }
 
+        if (guildMember.isOwner()) {
```

Comment:
Do we have a bot owner field? I could not find one in the JDA docs. Guild owner is the closest I got. If we want bot owner we need to store it ourselves.

Reply:
BotOwner is on the JDACommandManager

Reply:
make getBotOwnerId public, dunno why thats private

Reply:
Also this change is not valid. 

A command may be restricted the bots owner (for a bot that is shared between guilds), but you dont want the guilds owner to access it.

So remove this and simply update line 24 to return if the event user (not guild member) is the owner.

I'm not sure if that getGuild member is valid in context of a PM either.

Reply:
Yeah that's why I was hesitant because that would give guild owners * perms, but in the case of sharded bots that's no bueno.